components/Chips/docs/theming-extension.md

```diff
@@ -0,0 +1,37 @@
+### Theming Extensions
```

Comment:
Rename this file and it’s header to “Theming”.

Reply:
Addressed in 1823e91 and e0437b0.